examples/x86_64.swiftinterface

```diff
@@ -0,0 +1,80 @@
+@objc @objcMembers public class CommunicationUserIdentifier : ObjectiveC.NSObject, AzureCommunication.CommunicationIdentifier {
+  @objc final public let identifier: Swift.String
+  @objc(initWithIdentifier:) public init(_ identifier: Swift.String)
+  @objc deinit
+  @objc override dynamic public init()
+}
+@objc @objcMembers public class UnknownIdentifier : ObjectiveC.NSObject, AzureCommunication.CommunicationIdentifier {
+  @objc final public let identifier: Swift.String
+  @objc(initWithIdentifier:) public init(_ identifier: Swift.String)
+  @objc deinit
+  @objc override dynamic public init()
+}
+@objc @objcMembers public class PhoneNumberIdentifier : ObjectiveC.NSObject, AzureCommunication.CommunicationIdentifier {
+  @objc final public let phoneNumber: Swift.String
+  @objc final public let rawId: Swift.String?
+  @objc public init(phoneNumber: Swift.String, rawId: Swift.String? = nil)
+  public static func == (lhs: AzureCommunication.PhoneNumberIdentifier, rhs: AzureCommunication.PhoneNumberIdentifier) -> Swift.Bool
+  @objc deinit
+  @objc override dynamic public init()
+}
+@objc @objcMembers public class MicrosoftTeamsUserIdentifier : ObjectiveC.NSObject, AzureCommunication.CommunicationIdentifier {
+  @objc final public let userId: Swift.String
+  @objc final public let isAnonymous: Swift.Bool
+  @objc final public let rawId: Swift.String?
+  @objc final public let cloudEnviroment: AzureCommunication.CommunicationCloudEnvironment
+  @objc public init(userId: Swift.String, isAnonymous: Swift.Bool = false, rawId: Swift.String? = nil, cloudEnvironment: AzureCommunication.CommunicationCloudEnvironment = .Public)
+  @objc public init(userId: Swift.String, isAnonymous: Swift.Bool)
+  public static func == (lhs: AzureCommunication.MicrosoftTeamsUserIdentifier, rhs: AzureCommunication.MicrosoftTeamsUserIdentifier) -> Swift.Bool
+  @objc deinit
+  @objc override dynamic public init()
+}
+@objc @objcMembers public class CommunicationAccessToken : ObjectiveC.NSObject {
+  @objc final public let token: Swift.String
+  @objc final public let expiresOn: Foundation.Date
+  @objc public init(token: Swift.String, expiresOn: Foundation.Date)
+  @objc deinit
+  @objc override dynamic public init()
+}
+@objc @objcMembers public class CommunicationCloudEnvironment : ObjectiveC.NSObject {
+  @objc public static var Public: AzureCommunication.CommunicationCloudEnvironment
+  @objc public static var Dod: AzureCommunication.CommunicationCloudEnvironment
+  @objc public static var Gcch: AzureCommunication.CommunicationCloudEnvironment
+  @objc public init(environmentValue: Swift.String)
+  @objc public func getEnvironmentValue() -> Swift.String
+  public static func == (lhs: AzureCommunication.CommunicationCloudEnvironment, rhs: AzureCommunication.CommunicationCloudEnvironment) -> Swift.Bool
+  @objc deinit
+  @objc override dynamic public init()
+}
+public typealias TokenRefresherClosure = (@escaping AzureCommunication.TokenRefreshOnCompletion) -> Swift.Void
```

Comment:
`TokenRefreshOnCompletion` implies this should be called `TokenRefreshHandler` to conform with the guidlines:
https://azure.github.io/azure-sdk/ios_design.html#ios-network-callback-suffix

Reply:
@JoshuaLai I do wonder if `TokenRefreshHandler` is too vague. Perhaps `TokenRefreshCompletionHandler`? I'm okay with either.

Also, I would recommend that since you have a method that handles the refreshing and a callback that happens when a token is refreshed that you consider adding something like a `TokenRefreshDelegate` for developers that prefer that pattern. I would suspect many ObjC developers would be more familiar with it and prefer it. 
https://azure.github.io/azure-sdk/ios_design.html#delegates

This is merely a suggestion for future work. It would be a non-breaking additive change.